frontend/src/views/admin/matching/import-export/import-guarantees.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { Modal, Button, Row, Form, Col, Alert } from "react-bootstrap";
+import { DataFormat } from "../../../../libs/import-export";
+import { AppointmentGuaranteeStatus } from "../types";
+import { batchUpsertGuarantees } from "../actions";
+import { useThunkDispatch } from "../../../../libs/thunk-dispatch";
+
+/**
+ * A button that displays a modal to allow users to upload JSON files
+ * with information about subsequent appointment guarantees to be
```

Comment:
Confirming that the request here is to edit the docstring so that "upserted" is not on its own line, or did I misunderstand?

Made a commit with this change. Please mark as resolved if this is the correct interpretation.